expfmt/text_parse.go

```diff
@@ -34,6 +34,11 @@ import (
 // by nil.
 type stateFn func() stateFn
 
+const (
+	KeywordHelp = "HELP"
+	KeywordType = "TYPE"
```

Comment:
Could you lower-case those? It's fine to make "magic strings" constants, but I'd prefer if they don't show up in godoc (unless, of course, they are meant for external consumption).